abstract-document/src/main/java/com/iluwatar/abstractdocument/module-info.java

```diff
@@ -0,0 +1,2 @@
+module abstractdocument.modules {
+}
```

Comment:
I would prefer format like this:
```
module com.iluwatar.abstractdocument {
}

```

Reply:
The build is failing with `package org.slf4j is not visible
  (package org.slf4j is declared in the unnamed module, but module org.slf4j does not read it)`

To fix it you need to declare something like this inside the module-info:
`requires org.slf4j;`

Reply:
@iluwatar Thank you for your comments. I have done the required changes. Can you please suggest?